### PR TITLE
We don't need to use arch now

### PR DESCRIPTION
**Pull request by yamaguchi1024**

### CODE REVIEW

**Comment by yamaguchi1024** [comment]
Do you think this is good?

File: CMakeLists.txt [hunk 1]
```diff
@@ -40,6 +40,9 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
 
 include_directories(inc)
 
+message(STATUS "PYTHON_LIBRARIES = ${PYTHON_LIBRARIES}")
+message(STATUS "PYTHON_EXECUTABLE = ${PYTHON_EXECUTABLE}")
+message(STATUS "PYTHON_INCLUDE_DIRS = ${PYTHON_INCLUDE_DIRS}")
```
Comment (Teemperor) [inline CMakeLists.txt]: please ditch this (wrong?) debug code

**Comment by Teemperor** [comment]
Thanks! Some points:

1. Maybe squash the 22 commits into one with a short commit message.
2. Why don't we keep the Arch builds along the old/new ubuntu builds?

**Comment by yamaguchi1024** [comment]
I want to use docker if we use archlinux because Travis doesn't officially provide archlinux support and we have to use third-party script which is not very happy... Adding arch sounds like just extending compilation time to me when trusty works.

**Pull request merged**